src/sentry/testutils/symbolicator.py

```diff
@@ -0,0 +1,132 @@
+import os
+import zipfile
+from io import BytesIO
+from unittest.mock import patch
+
+import pytest
+from django.core.files.uploadedfile import SimpleUploadedFile
+from django.urls import reverse
+
+from sentry.utils.safe import get_path
+
+from .cases import TransactionTestCase
+from .relay import RelayStoreHelper
+
+
+def strip_frame(frame):
+    if frame:
+        frame = {
+            "data": {
+                "symbolicator_status": get_path(frame, "data", "symbolicator_status"),
+                "orig_in_app": get_path(frame, "data", "orig_in_app"),
+            },
+            "function": frame.get("function"),
+            "instruction_addr": frame.get("instruction_addr"),
+            "symbol": frame.get("symbol"),
+            "package": frame.get("package"),
+            "lineno": frame.get("lineno"),
+            "in_app": frame.get("in_app"),
+            "trust": frame.get("trust"),
+        }
+
+    return frame
+
+
+def strip_stacktrace(stacktrace):
+    if stacktrace:
+        stacktrace = dict(stacktrace)
+        stacktrace["frames"] = [strip_frame(x) for x in stacktrace.get("frames") or ()]
+
+    return stacktrace
+
+
+def strip_stacktrace_container(container):
+    if container:
+        container = dict(container)
+        container["stacktrace"] = strip_stacktrace(container.get("stacktrace"))
+        container["raw_stacktrace"] = strip_stacktrace(container.get("raw_stacktrace"))
+
+    return container
+
+
+@pytest.fixture(scope="function")
+def live_server_cls(request, live_server):
+    if hasattr(request, "cls"):
+        request.cls.live_server = live_server
+
+
+@pytest.mark.usefixtures("live_server_cls")
+class SymbolicatorTestCase(RelayStoreHelper, TransactionTestCase):
```

Comment:
FWIW I've always recommended not to mix fixtures and unittest-style tests, the interactions aren't very well defined by pytest.  It is better to move everything over to fixtures.

Reply:
fwiw i agree - trying to undo the use of them here is a bit tricky particularly because of `live_server`. i plan to get back to this but im still fixing wsl at home and havent been back to the office yet where i can code :)

Reply:
taking a new approach - still relies on fixtures because we have to, but im trying to override the live_server fixture and _always_ set the appropriate behavior here, vs doing this as a one-off in symbolicator

i dont think its yet working, but since i work on multiple machines ive pushed up the latest attempt. im not familiar w/ this service so its taking me longer to diagnose whats going on than id like